docs/release-notes/v12.1.0.md

```diff
@@ -0,0 +1,66 @@
+---
+id: v12.1.0
+title: Version 12.1.0 Release Notes
+sidebar_label: v12.1.0
+image: "img/release-note-link-preview.png"
+---
+
+_January 18th, 2023_
+
+
+:::info
+Mia-Platform Console v12.1.0 is **now in Preview!**
+
+Console SaaS users can try out v12.1.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## Console
+
+### New Features
+
+#### New User events for the Webhook section
+
+User events have been added that will allow a Company Owner to configure and trigger specific webhooks when user access management changes occur.
+
+Listed below are the events that have been added with this release:
+
+* User added to Company
+* User Company role edited.
+* User removed from Company
+* User added to Project
+* User removed from Project
+
+For more information regarding individual events, please [go to the documentation]().
+
+#### Complete company access management with the CLI (miactl)
+
+The new version of miactl allows complete management of all User Company access management.
+In particular from today it will be possible to add, modify and remove any type of identity (User, Service Account or Group) simply through command line.
+
+For more information on individual commands, please [go to the documentation]().
```

Comment:
```suggestion
#### Complete Company access management through `miactl`

The new version of `miactl` allows complete management of all identities access to the Company.
More specifically, starting from today it will be possible to add, modify and remove any type of identity (User, Service Account, Group) simply through command line.

For more information on the specific commands introduced, please [go to the documentation]().
```